patch_docs.py

```
@@ -0,0 +1,103 @@
+        f" *   {method}:",
+        f" *     summary: Endpoint for {method.upper()} {openapi_path}",
+        f" *     tags: [{tag}]",
+        " *     security:",
+        " *       - cookieAuth: []",
+        " *       - bearerAuth: []"
+    ]
+    
+    if params:
+        doc.append(" *     parameters:")
+        for p in params:
+            doc.append(f" *       - in: path")
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)